src/api/java/io/github/cvc5/api/Solver.java

```diff
@@ -1555,6 +1558,8 @@ private native long declareFun(
    * {@code
    *   ( declare-sort <symbol> <numeral> )
    * }
+   * @apiNote This corresponds to mkUninterpretedSort() if arity = 0, and to
+   *          mkUninterpretedSortConstructorSort() if arity > 0.
```

Comment:
```suggestion
   *          mkUninterpretedSortConstructorSort(arity) if arity > 0.
```

Reply:
We can't use the types of the arguments in the java doc, so I'll leave it as it is and have only fixed the problem with `>`.